source/geod24/LocalRest.d

```diff
@@ -303,22 +303,54 @@ public void sleep (Duration timeout)
         dg = If non-null, this delegate will be called when the timer fires
         periodic = Speficies if the timer fires repeatedly or only once
 
+    Returns:
+        a Timer instance with the ability to stop the timer
+
 *******************************************************************************/
 
-public void setTimer (Duration timeout, void delegate() dg,
+public Timer setTimer (Duration timeout, void delegate() dg,
     bool periodic = false)
 {
     assert(scheduler !is null, "Cannot call this delegate from the main thread");
     assert(dg !is null, "Cannot call this delegate if null");
-    scheduler.schedule(
-    ()
+
+    Timer timer = Timer(timeout, dg, periodic);
```

Comment:
I think we discussed storing those in the scheduler, didn't we ?

Reply:
I didn't understand the benefits of storing Timer in the scheduler.

Are you saying that It assigns one Timer to one scheduler?
Or does this control Timer in the schedule?

Reply:
I think that was the alternative implementation idea.